static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -167,6 +169,9 @@ export function Content({
 
     const platform = getStacktracePlatform(event, exc.stacktrace);
 
+    // The banners should appear on the top frame only
+    const isTopFrame = newestFirst ? excIdx === values.length - 1 : excIdx === 0;
```

Comment:
Nit: not really a frame, more like the top exception

Reply:
yeah good point